src/WebSdk/Publish/Targets/Microsoft.NET.Sdk.Publish.props

```diff
@@ -11,4 +11,11 @@ Copyright (c) .NET Foundation. All rights reserved.
 -->
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
+  <PropertyGroup>
+    <EnableSdkContainerSupport Condition="'$(EnableSdkContainerSupport)' == ''">true</EnableSdkContainerSupport>
+  </PropertyGroup>
+
+  <Import Project="$(MSBuildThisFileDirectory)..\..\..\Containers\build\Microsoft.NET.Build.Containers.props"
```

Comment:
The layout of the file (to out of the sdk path and find this) is odd but I understand we may not have a better way to deal with it. If there was a sdksroot property or something, we could avoid unintentionally breaking things if we decide to move this file somewhere else.